package exception;

public class UnCheckedException {

	public static void main(String[] args) {
		System.out.println(args[0]);
		//ArrayIndexOutOfBoundsException :0번 자체가 없다 어쩌고
	} //런타임 돌려봐야 알 수 있다 

}
